depends/packages/bdb.mk

```diff
@@ -8,6 +8,7 @@ $(package)_build_subdir=build_unix
 
 define $(package)_set_vars
   $(package)_config_opts=--disable-shared --enable-cxx --disable-replication
+  $(package)_config_opts+=--libdir=$($($(package)_type)_prefix)/lib
```

Comment:
Out of curiosity, what does `$($($(package)_type)_prefix)` expand to?

Reply:
```
$(package)_type=$(host_arch)_$(host_os)
$(host_arch)_$(host_os)_prefix=$(BASEDIR)/$(host)
```

Reply:
And more specifically, from line 62 of funcs.mk...

$(1)_prefixbin:=$($($(1)_type)_prefix)/bin/ ....

So.... $($($(package)_type)_prefix)/lib forces the packages to store their libs and pkgconfig in lib instead of lib64, which is the tendency on Redhat and openSuSE systems when left to their own devices.